README.md

```diff
@@ -46,7 +46,69 @@ ActiveRecord models that use this connection will now be connecting to the confi
 
 ## Testing
 
-To run the tests, you'll need the ODBC driver as well as the connection adapter for each database against which you're trying to test. Then run `DSN=MyDatabaseDSN bundle exec rake test` and the test suite will be run by connecting to your database.
+To run the tests, you'll need the ODBC driver as well as the connection adapter for each database against which you're trying to test. You'll then need to set either the `CONN_STR` or `DSN` env var to point at the database and driver you're using.
+
+1. Install ruby 3.2
+
+   - If you don't already have ruby 3.2 installed locally, one option is using rbenv via the following:
+
+      1. Install [rbenv](https://github.com/rbenv/rbenv?tab=readme-ov-file#homebrew) via homebrew
+      2. `rbenv install 3.2.0`
+      3. `rbenv local 3.2.0`
+
+   - Note: if you are using nix for local development of other apps, the version of libyaml installed by this method can interfere with the building of native extensions for those apps. To fix that, you'll need to run `brew uninstall --ignore-dependencies libyaml` and re-run your bundle install inside of that nix shell. You'll then need to reinstall libyaml via homebrew if you want to work on this gem.
+
+2. Run `bundle install` in this directory to install the relevant gems
```

Comment:
This isn't going to work verbatim because...OS differences, but...CI is seeing the same error I was locally that this resolves.

Weird that it wasn't happening before though.

Reply:
> Do you happen to know anything about how the bundler cache works here?

Does [this](https://github.com/ruby/setup-ruby?tab=readme-ov-file#caching-bundle-install-automatically) answer your questions?